package.json

```diff
@@ -21,8 +22,18 @@
     "url": "https://github.com/huafu/hubot-gitter2/issues"
   },
   "homepage": "https://github.com/huafu/hubot-gitter2",
+  "files": [
+    "/doc/",
```

Comment:
I included this when removing __.npmignore__ to avoid changing the list of files included in the tarball. Actually, though, I suggest we exclude this HTML documentation from the tarball. Do you agree, @huafu?

Is the HTML documentation up to date? If so, how is it updated? If not, shall we remove it from the repository?

Reply:
The doc is outdated I guess, anyway:
- yes, remove `doc` from the release since it should be viewable online only
- if I remember well there was a script generating the doc from the comments and then publishing it to the `gh-pages` branch, I'll try to take a look at that and give you more info...

Reply:
I removed `"/doc/"` from `"files"`. :)